zrml/swaps/src/lib.rs

```diff
@@ -1192,6 +1197,17 @@ mod pallet {
                 .cloned()
                 .ok_or(Error::<T>::AssetNotBound)
         }
+
+        /// Calculate the exit fee percentage for `pool`.
+        fn _calc_exit_fee(pool: &Pool<BalanceOf<T>, T::MarketId>) -> BalanceOf<T> {
```

Comment:
Just wondering what the underscore before the function is for? I know in python this can be used to hint that the function is private, in rust a non-pub function is inherently private though.

Reply:
Yeah, I was inspired by [this](https://docs.substrate.io/how-to-guides/v3/basics/helper-functions/) and my background in Python. But honestly, I don't see the point anymore. I'll remove the underscore (it's also inconsistent wiht the other helper functions).